src/Mautic/Config.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Escopecz\MauticFormSubmit\Mautic;
+
+/**
+ * Mautic Configuration
+ */
+class Config
+{
+    /**
+     * Curl verbose logging option
+     *
+     * @var int
+     */
+    protected $curlVerbose; 
+    
+    /**
+     * Constructor
+     *
+     * @param int $curlVerbose
+     */
+    public function __construct(int $curlVerbose)
```

Comment:
I'd remove the constructor as the config DTO can grow into tens of options. We shouldn't require to set all of them in the constructor.